compiler/src/dotty/tools/dotc/quoted/Runners.scala

```diff
@@ -31,6 +31,7 @@ object Runners {
   }
 
   case class RunSettings(
-    optimise: Boolean = false
+    optimise: Boolean = false,
+    outDir: Option[String] = None
```

Comment:
outputDir, also document the meaning of Some/None here.

Reply:
You can do `java -Djdk.internal.lambda.dumpProxyClasses=out` to be able to debug the classes generated for lambda calls at runtime. Maybe we should have a `-Ddotty.meta.dumpClasses` or something for similar purpose.